doc(ocs): Improved doc to expose Capabilities

zak39 opened the pull request:
### ☑️ Resolves

I've added more context about where we create Capabilities and how to register it.

zak39 committed:
doc(ocs): Improved doc to expose Capabilities

I've added more context about where we create Capabilities and how to
register it.

susnux commented on developer_manual/client_apis/OCS/ocs-openapi.rst:
```diff
@@ -698,6 +698,7 @@ How to expose Capabilities
 --------------------------
 
 Imagine we take the same Todo app of the previous example and want to expose some capabilities to let clients know what they can expect.
+First, you should create your `Capabilities` in the `lib/AppInfo` folder :
```
Why there? In most cases I saw it in the lib root

Copilot committed:
doc(ocs): fix Capabilities.php location to lib folder

Agent-Logs-Url: https://github.com/nextcloud/documentation/sessions/573aae94-62bc-43a0-8545-9f37b38e5fea

Co-authored-by: nextcloud-command <88102737+nextcloud-command@users.noreply.github.com>

skjnldsv commented on developer_manual/client_apis/OCS/ocs-openapi.rst:
```diff
@@ -841,8 +734,74 @@ Now you have to add the correct return type annotation:
+
+    namespace OCA\Todo\AppInfo;
+
+    use OCP\AppFramework\Bootstrap\IBootstrap;
```
```suggestion
    use OCA\Todo\Capabilities;
    use OCP\AppFramework\Bootstrap\IBootstrap;
```

skjnldsv committed:
fix(developer): missing `Capabilities` import

Signed-off-by: John Molakvoæ <skjnldsv@users.noreply.github.com>

nextcloud-command commented:
@copilot resolve the merge conflicts in this pull request

Copilot committed:
Merge remote-tracking branch 'origin/master' into doc/expose-capabilities

# Conflicts:
#	developer_manual/client_apis/OCS/ocs-openapi.rst

Co-authored-by: nextcloud-command <88102737+nextcloud-command@users.noreply.github.com>
